gfx/map/map_object_data/building_locators.txt

```diff
@@ -1759,7 +1759,7 @@
 		}
 		{
 			id=294
```

Comment:
If you change the heightmap don't leave it flat. Make it pretty.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133262300-44796e5b-9c48-48d3-8433-82fc708b0b2f.png)

</details>

Reply:
Fixed in 71bfe27. Pending review.